detections/endpoint/suspicious_local_llm_framework_download_and_execution_via_sysmon.yml

```diff
@@ -0,0 +1,67 @@
+name: Suspicious Local LLM Framework Download and Execution via Sysmon
```

Comment:
this detection is same as above, consider adding this test data to the above detection and remove this yaml

Reply:
I tried to combine both and i could not maybe you can take another look